### DISCUSSION TITLE
Setup Guide For OSX?

### DISCUSSION THREAD

**Discussion by tekn0x** [Eugeny/tabby#7442] [Q&A]
I thought that tabby looked absolutely beautiful from all the screenshots. Once I installed it on my Mac, it literally looks the same as a normal terminal and nothing like the screenshots. Fonts have no colors or icons, prompt is plain Jane.

I have googled around and unfortunately not been able to find any tutorials or guides on how to set this up. I thought for some reason these beautiful settings were the default but apparently not?

Could anyone recommend a guide on how to get it looking like it does in the screenshots with the cool fonts icons, the really cool looking prompt. Colored output on mac?

Thank you!

**Comment by Eugeny** [comment]
Hi, it's just oh-my-zsh with default settings.
